cypher/cypher-docs/src/docs/dev/ql/administration/security/grant-set-property-syntax.asciidoc

```diff
@@ -1,8 +1,8 @@
 .Command syntax
 [source, cypher]
 -----
-GRANT SET PROPERTY "{" { * | property-name[, ...] } "}"
-    ON GRAPH[S] { * | graph-name[, ...] }
+GRANT SET PROPERTY "{" { * | property[, ...] } "}"
```

Comment:
I think we should either leave `property-name`, or change the other elements' names, e.g., `label-name`, `label-or-rel-type-name`.

Reply:
I could go over and update `label-name`, `rel-type-name` and `label-or-rel-type-name` to not have name as well and just be `label`, `rel-type` and `label-or-rel-type`, but since that have to be done for all versions I think that will make more sense as new PRs

Reply:
added [a trello card](https://trello.com/c/2G1PP9KS/2302-more-naming-clean-up-in-docs) to update these places as well

Reply:
Yes, thanks!